model/test_results.go

```diff
@@ -649,52 +649,18 @@ func (t TestResult) convertToParquet() ParquetTestResult {
 // TestResultsTaskOptions specify the criteria for querying test results by
 // task.
 type TestResultsTaskOptions struct {
-	TaskID string
-	// TODO (EVG-18798): Make this field required once Evergreen and Spruce
-	// are updated.
-	Execution *int
-	// TODO (EVG-18798): Remove this field once Evergreen and Spruce are
-	// updated.
-	DisplayTask bool
+	TaskID    string
+	Execution int
 }
 
 func (opts *TestResultsTaskOptions) createFindOptions() *options.FindOptions {
-	findOpts := options.Find().SetSort(bson.D{{Key: bsonutil.GetDottedKeyName(testResultsInfoKey, testResultsInfoExecutionKey), Value: -1}})
-	if !opts.DisplayTask {
-		findOpts = findOpts.SetLimit(1)
-	}
-
-	return findOpts
+	return options.Find().SetSort(bson.D{{Key: bsonutil.GetDottedKeyName(testResultsInfoKey, testResultsInfoExecutionKey), Value: -1}})
```

Comment:
Given the execution will be given to us explicitly do we still need to do a sort?

Reply:
Good catch!